File: Pratica 6 - Funcoes 2/python_logic/exercicio7_v2.py
def classificar_nota(nota):
    if nota >=90 and nota <= 100:
        print("A")
    elif nota >= 80 and nota < 90:
        print("B")
    elif nota >= 70 and nota < 80:
        print("C")
    elif nota >= 60 and nota < 70:
        print("D")
    elif nota >= 0 and nota < 60:
        print("E")
    else:
        print("Nota inválida.")

nota = int(input("Digite uma nota entre 0 e 100: "))
classificar_nota(nota)

File: Pratica 6 - Funcoes 2/java_logic/Exercicio7_v2.java
import java.util.Scanner;

public class Exercicio7_v2 {
    static String classificar(int nota) {
        if (nota >= 90 && nota <= 100) return "A";
        else if (nota >= 80 && nota < 90) return "B";
        else if (nota >= 70 && nota < 80) return "C";
        else if (nota >= 60 && nota < 70) return "D";
        else if (nota >= 0 && nota < 60) return "E";
        else return "Nota inválida!";
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite uma nota de 0 a 100: ");
        int nota = sc.nextInt();
        System.out.println(classificar(nota));

        sc.close();
    }
}
